kong.yml

```diff
@@ -90,7 +91,7 @@ nginx: |
     real_ip_recursive on;
 
     # Other Settings
-    client_max_body_size 128m;
+    client_max_body_size 0m;
```

Comment:
Ok its 1 mb. But  if we set a max payload size, plugin would be useless beyond that size limit. I thought whole idea for this pluingin is to give control to user. We do have to mention in our doc that  by default this field is disabled.